shopfloor_mobile_base/static/wms/src/components/searchbar/searchbar.js

```diff
@@ -52,6 +52,11 @@ export var Searchbar = Vue.component("searchbar", {
             type: Number,
             default: 0,
         },
+        // uses the clearable input type logic (using an X icon to clear the input)
+        clearable_input_type: {
+            type: Boolean,
+            default: false,
```

Comment:
can be true likely?

Reply:
This is a very specific thing of this filter searchbar (as opposed to the standard scan searchbar we use everywhere), I wouldn't set it as true by default as it can mess with the rest of the searchbars.

I've given it a try and it alters the behaviour of the rest of the searchbars in the app.